mosaic-site/src/components/mdx/h4/Heading4.tsx

```diff
@@ -1,6 +1,13 @@
-import { ReactNode } from "react";
+import { FC, ReactNode } from "react";
 import styles from "./Heading4.module.css";
 
-export const Heading4 = ({ children }: { children: ReactNode }) => (
-  <h4 className={styles.heading4}>{children}</h4>
+type Heading4Props = {
```

Comment:
these can probably be done via a theme contract .. avoiding the need to have a separate component.
One we can loop back on as we start to converge theming

Reply:
yeah that would be good :)